content/en/docs/ibm/existing-cluster.md

```diff
@@ -4,15 +4,11 @@ description = "Set up a cluster if you already have one"
 weight = 5
                     
 +++
-{{% alert title="Out of date" color="warning" %}}
-This guide contains outdated information pertaining to Kubeflow 1.0. This guide
-needs to be updated for Kubeflow 1.1.
-{{% /alert %}}
 
 ## Initial Setup for Existing Cluster
 
 Get the Kubeconfig file:
 
 	ibmcloud ks cluster config --cluster $CLUSTER_NAME
```

Comment:
Also, let's add command line/code blocks here for consistency:

```suggestion
	```shell
	ibmcloud ks cluster config --cluster $CLUSTER_NAME
	```
```

What do you think? @adrian555 Thank you!